src/platform/backends/shared/linux/backend_utils.cpp

```diff
@@ -266,6 +267,17 @@ std::string mp::Backend::get_subnet(const mp::Path& network_dir, const QString&
     return new_subnet;
 }
 
+std::vector<mp::NetworkInterfaceInfo>::const_iterator mp::Backend::find_bridge_with(
```

Comment:
There is some duplicate happening between here and [here](https://github.com/canonical/multipass/blob/71d73f38f0724c5899ad68b01919eae686c87ba2/src/platform/backends/shared/base_virtual_machine_factory.cpp#L35). I see the duplicate implementation being modified in #3513, but it would be nice to see these combined into a common implementation in the this utils file.

Reply:
Right but, in any case, this is outdated.

Reply:
Ok, this is fine now, but...